### PR TITLE
Oneof -> inheritance

### PR DESCRIPTION
**Pull request by garrettjonesgoogle**

### CODE REVIEW

**Comment by garrettjonesgoogle** [comment]
Moved factory methods into Any* classes - PTAL

File: update_baselines.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
```
Comment (michaelbausor) [inline update_baselines.sh]: Was it intentional to check this in? It seems quite useful.

Comment (garrettjonesgoogle) [reply update_baselines.sh]: yes, and it was very easy :-)

**Comment by garrettjonesgoogle** [comment]
Superseded by #26

**Pull request closed without merging**